tests/main.test.ts

```diff
@@ -112,6 +112,26 @@ describe("User start/stop", () => {
     );
   });
 
+  test("Author's manual unassign should close linked issue", async () => {
+    const infoSpy = jest.spyOn(console, "info").mockImplementation(() => {});
+    const issue = db.issue.findFirst({ where: { id: { equals: 2 } } }) as unknown as Issue;
+    const sender = db.users.findFirst({ where: { id: { equals: 2 } } }) as unknown as PayloadSender;
+    const context = createContext(issue, sender, "") as Context<"issues.unassigned">;
+
+    context.adapters = createAdapters(getSupabase(), context);
+
+    const { content } = await userUnassigned(context);
+
+    expect(content).toEqual("Linked pull-requests closed.");
+    const logs = infoSpy.mock.calls.flat();
+    expect(logs[0]).toMatch(/Opened prs/);
+    expect(cleanLogString(logs[3])).toMatch(
+      cleanLogString(
+        " › ```diff# These linked pull requests are closed:  http://github.com/ubiquity/test-repo/pull/2  http://github.com/ubiquity/test-repo/pull/3"
```

Comment:
I think this message is unnecessary. Please remove and then you can merge this.

Reply:
Removed in https://github.com/ubiquibot/command-start-stop/pull/46/commits/cc9e891624d584a487f8bde1367f4805b1937333. The test is still valid because we log that message to the console.